src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -300,5 +307,1772 @@ private static string ConvertIcuTimeFormatString(string icuFormatString)
 
             return StringBuilderCache.GetStringAndRelease(sb);
         }
+        
+        private static string LCIDToLocaleName(int culture)
+        {
+            string cultureName;
+            if (LcidToCultureNameMapper.TryGetValue(culture, out cultureName))
+            {
+                return cultureName;
+            }
+            
+            return null;
+        }
+
+        private static int LocaleNameToLCID(string cultureName)
+        {
+            int lcid;
+            if (CultureNameToLcidMapper.TryGetValue(AnsiToLower(cultureName), out lcid))
+            {
+                return lcid;
+            }
+            
+            return LOCALE_CUSTOM_UNSPECIFIED;
+        }
+
+        private static CultureNameToLcidDictionary CultureNameToLcidMapper
+        {
+            get
+            {
+                if (s_cultureNameToLcidDictionary == null)
+                {
+                    CultureNameToLcidDictionary tempMapping = new CultureNameToLcidDictionary()
+                    {
+                        { "", 0x0007f },
```

Comment:
Would it make sense to do this table in more compact way like we are doing it elsewhere? 

E.g. [this](https://github.com/dotnet/corert/blob/26413145abcbdd49468db230a6f29784494f6f46/src/System.Private.CoreLib/src/System/Text/EncodingTable.cs#L46) should match almost exactly.